src/Nethermind/Nethermind.JsonRpc.Test/Modules/SubscribeModuleTests.cs

```diff
@@ -805,6 +804,43 @@ public void NewPendingTransactionsSubscription_on_NewPending_with_includeTransac
             expectedResult.Should().Be(serialized);
         }
 
+        [TestCase(2)]
+        [TestCase(5)]
+        [TestCase(10)]
+        [Explicit("Requires a WS server running")]
+        public async Task NewPendingTransactionSubscription_multiple_fast_messages(int messages)
```

Comment:
Can you make test with competing subscriptions (same/different channel)

Reply:
Added test and it fails: since each subscription handles it's own queue of tasks we end up with the same problem.